### PR TITLE
Add setup.py to create a package

### PR DESCRIPTION
**Pull request by kerel-fs**
Fixes glidernet/ogn-python#16.
[WIP]

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    long_description = f.read()
+
+setup(
+    name='ogn-python',
+    version=PACKAGE_VERSION,
+    description='A python framework for the Open Glider Network (OGN)',
+    long_description=long_description,
+    url='https://github.com/glidernet/ogn-python',
+
+    author='Konstantin Gründger aka Meistschueler, Fabian P. Schmidt aka kerel',
```
Comment (Meisterschueler) [inline setup.py]: "Meisterschueler" instead of "Meistschueler"

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    author_email='kerel-fs@gmx.de',
+
+    license='AGPLv3',
+
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'Topic :: Software Development :: Build Tools',
```
Comment (Meisterschueler) [inline setup.py]: it is not a build tool. What about: 'Topic :: Scientific/Engineering :: GIS',

File: setup.py (hunk 1 above)
Comment (kerel-fs) [inline setup.py]: fixed.

File: setup.py (hunk 2 above)
Comment (kerel-fs) [inline setup.py]: good, incorporated it.

**Pull request merged**